src/main/java/org/tugraz/sysds/runtime/lineage/LineageCache.java

```diff
@@ -138,8 +149,8 @@ public static synchronized MatrixBlock get(Instruction inst, LineageItem key) {
 	
 	public static boolean isReusable (Instruction inst) {
 		// TODO: Move this to the new class LineageCacheConfig and extend
-		return (inst.getOpcode().equalsIgnoreCase("tsmm")
-			|| inst.getOpcode().equalsIgnoreCase("ba+*"));
+		return (inst.getOpcode().equalsIgnoreCase("tsmm"));
+			//|| inst.getOpcode().equalsIgnoreCase("ba+*"));
```

Comment:
Why does this PR disable full reuse for matrix multiplication - please make TODO and say why you need to disable it.